GiniVision/Classes/Core/CameraViewController.swift

```diff
@@ -614,15 +651,53 @@ extension CameraViewController {
     }
 }
 
+// MARK: - DocumentPickerCoordinatorDelegate
+
+extension CameraViewController: DocumentPickerCoordinatorDelegate {
+
+    func documentPicker(_ coordinator: DocumentPickerCoordinator,
+                        didPick documents: [GiniVisionDocument],
+                        from: DocumentPickerType,
+                        completion: DocumentPickerCompletion?) {
+        self.validate(importedDocuments: documents) { validatedDocuments in
+            if let firstElement = validatedDocuments.first,
```

Comment:
Why only check the first one?

Reply:
Nice catch, this is a bug. It should be first `firstElementWithError`